src/infi/clickhouse_orm/query.py

```diff
@@ -442,6 +452,14 @@ def order_by(self, *field_names):
         qs._order_by = field_names
         return qs
 
+    def having(self, having_str):
```

Comment:
what is `having_str` ?
are you proposing to provide a condition just as a string?
It's a bit changing the concept of ORM, unlike when we do `filter` there is kwargs approach

Reply:
I use Q function to spport having. Thanks to check it

Reply:
I use Q function to support HAVING. Thanks to check it